kolibri/plugins/coach/assets/src/views/reports/ReportsLearnerReportLessonPage.vue

```diff
@@ -65,7 +65,10 @@
               <StatusSimple :status="tableRow.statusObj.status" />
             </td>
             <td>
-              <TimeDuration :seconds="tableRow.statusObj.time_spent" />
+              <TimeDuration
+                v-if="tableRow.statusObj.status !== STATUSES.notStarted"
+                :seconds="tableRow.statusObj.time_spent"
```

Comment:
instead of hiding the component with `v-if` we should pass `undefined` into `:seconds`. This will cause it to show the gray dash